testing/README.md

```diff
@@ -793,3 +793,56 @@ Deno.test("randomMultiple uses randomInt to generate random multiples between -1
   assertSpyCalls(randomIntStub, 2);
 });
 ```
+
+### Faking time
+
+Say we have a function that has time based behavior that we would like to test.
+With real time, that could cause test's to take much longer than they should. If
+you fake time, you could simulate how your function would behave over time
+starting from any point in time. Below is an example where we want to test that
+the callback is called every second.
+
+```ts
+// https://deno.land/std@$STD_VERSION/testing/mock_examples/interval.ts
+export function secondInterval(cb: () => void): number {
+  return setInterval(cb, 1000);
+}
+```
+
+With `FakeTime` we can do that. When the `FakeTime` instance is created, it
+splits from real time. The Date, setTimeout, clearTimeout, setInterval, and
+clearInterval globals are replaced with versions that use the fake time until
+real time is restored. You can control how time ticks forward with the `tick`
+method on the `FakeTime` instance.
```

Comment:
```suggestion
With `FakeTime` we can do that. When the `FakeTime` instance is created, it
splits from real time. The `Date`, `setTimeout`, `clearTimeout`, `setInterval` and
`clearInterval` globals are replaced with versions that use the fake time until
real time is restored. You can control how time ticks forward with the `tick`
method on the `FakeTime` instance.
```

I don't understand what "splits" means in this sentence

Reply:
The suggestion caused lint to fail. I'll run deno fmt tonight when addressing the other feedback